src/components/Account/accounNavBar.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../styles/variables.scss";
+
+.NavBarAccount ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  height: 40px;
+  li {
+    color: $VeryLightPink;
+    padding: 8px;
```

Comment:
evita usar px, usa solo em/rem, por razones de accesibilidad (aunque para este proyecto no tengo una opinion muy fuerte)

Reply:
I take you advice. Thanks and in the next commit I will use em.